<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Students_MainWindow</class>
 <widget class="QMainWindow" name="Students_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>欢迎进入选课系统</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>当前学生信息：</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                 </font>
                </property>
                <property name="text">
                 <string>姓名：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_std_name">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5">
              <property name="spacing">
               <number>6</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <item>
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                 </font>
                </property>
                <property name="text">
                 <string>学号：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_std_num">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,5">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                 </font>
                </property>
                <property name="text">
                 <string>院系：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_std_college">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
             </font>
            </property>
            <property name="text">
             <string>已选课程：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget_selected_courses"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="pushButton_select">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                 </font>
                </property>
                <property name="text">
                 <string>选课</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_withdraw">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                 </font>
                </property>
                <property name="text">
                 <string>退课</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_return">
                <property name="font">
                 <font>
                  <family>Microsoft YaHei UI</family>
                 </font>
                </property>
                <property name="text">
                 <string>返回</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>50</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget_all_courses"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>帮助</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
